client/main/components/auth/ResetPassword.vue

```diff
@@ -26,9 +26,9 @@
       </validation-provider>
       <validation-provider
         v-slot="{ errors }"
+        :validate="{ rules: { required: true, is: password } }"
```

Comment:
Why not just import the missing `confirmed` rule within the validation file? 🤔
I think this is meant to be handled with that rule since this case is also the example in docs.